jsonnet/rules.libsonnet

```diff
@@ -548,6 +548,31 @@ function(params) {
         },
       ],
     },
+    {
+      name: 'openshift-etcd-telemetry.rules',
+      rules: [
+        {
+          expr: 'histogram_quantile(0.90, sum(rate(etcd_disk_wal_fsync_duration_seconds_bucket{job="etcd"}[5m])) by (instance, le))',
```

Comment:
do we want to use irate vs rate?

Reply:
As we are doing a sum here, we want to use rate, irate would not give us the right picture. But agreed overall for spikes irate is good when trying to represent it on a dashboard, but here we want a more accurate picture when sending via telemetry.

Reply:
thanks for explanation